### PR TITLE
DOP-2174: Card-ref styling

### PR DESCRIPTION
**Pull request by mango-db**
### Stories/Links:

[DOP-2174](https://jira.mongodb.org/browse/DOP-2174)
[index.txt rST](https://github.com/mango-db/docs-realm/blob/master/source/index.txt)
[Figma](https://www.figma.com/file/hPTwMb9uHOypYLckYDg072/Realm-PLP?node-id=0%3A1)

### Staging Links:

[Docs-realm](https://docs-mongodborg-staging.corp.mongodb.com/master/realm/cgoecknerwald/DOP-2174/)
[Docs-compass](https://docs-mongodborg-staging.corp.mongodb.com/master/compass/cgoecknerwald/DOP-2174/
) for sanity check <- nothing should break, though the breakpoint for cards to switch to 2 columns is bumped up to ~1200px

#### Tabs margin change

[Cloud-docs 1](https://docs-mongodbcom-staging.corp.mongodb.com/master/cloud-docs/cgoecknerwald/DOP-2174/security-add-mongodb-users/#open-the-add-new-database-user-dialog) <- no changes expected
[Cloud-docs 2](https://docs-mongodbcom-staging.corp.mongodb.com/master/cloud-docs/carolinao/DOP-1728/security-multi-factor-authentication/#set-up-an-authentication-method) <- no changes expected

### Notes:
- Hero image [handled separately](https://github.com/mongodb/snooty/pull/438)

### CODE REVIEW

**Review by allisonmui9 — COMMENTED** [review]
Looks great Claire!! Thank you! 
My only comment is that if we could get the tags to only span 2 lines that would be perfect. The cards look quick unbalanced when next to eachother. 

A suggestion: 
- I see there's some extra padding to the right of the margin looks like 1-2 px? Not sure where it's coming from. Could we remove that? 
- Could we drop the margin to 4px to see if that does anything? 

Potentially not part of this review, but the second card, is it possible to fix the link's margin to the paragraph to 16px? This may help with the unbalance issue!

File: src/components/Card.js [hunk 1]
```diff
@@ -19,12 +20,37 @@ const StyledCard = styled(LeafyGreenCard)`
   p:last-of-type {
     margin-bottom: 0;
   }
+
+  &:hover {
+    box-shadow: ${({ url }) => (url ? `` : `0 4px 10px -4px rgba(6,22,33,0.3)`)};
```
Comment (rayangler) [inline src/components/Card.js:25]: I can't seem to see this box shadow (nor the original box shadow) on hover on either of the cards on the staging links. Is this expected?

Comment (mango-db) [reply src/components/Card.js:25]: Odd - no, looking into it!

**Review by rayangler — APPROVED** [review]
🚀

**Review by schmalliso — APPROVED** [review]
Approved pending @allisonmui9's confirmation that it all looks good to her!

**Review by allisonmui9 — APPROVED** [review]
lgtm!

**Pull request merged**